cmd/options_test.go

```diff
@@ -56,20 +57,19 @@ func TestGetOptions(t *testing.T) {
 		}, additionalArgs...)
 
 		if withServerOptions {
-			args = append(args, "-"+endpointFlagName+"="+endpoint)
+			args = append(args, "--"+endpointFlagName+"="+endpoint)
```

Comment:
Do the previous versions not work anymore? Do we care about this theoretically breaking change?

Reply:
To provide some context, this change was introduced as a result of migrating to the `pflag` package which is a "drop-in replacement for Go's flag package". This flag type is used to add the component-based flags: https://pkg.go.dev/k8s.io/component-base/logs/api/v1#AddFlags.

Unlike Go's standard flag package, when using the `pflag` package, a single dash before an option means something different than a double dash. Single dashes signify a series of shorthand letters for flags. Hence this change.

The new behavior for using the "shorthand flags" `-` is as follows:
```
$ ./aws-ebs-csi-driver -aws-sdk-debug-log=true

unknown shorthand flag: 'a' in -aws-sdk-debug-log=true
```

It is common convention to use a single hyphen `-` to specify a single character flag, and a double hyphen `--` to specify a keyword flag so I don't believe this change is particularly dangerous but if it is a concern we could theoretically support shorthand flags as well.

Reply:
I think this is an acceptable justification, but we MUST make sure to call this out in the changelog.